src/routes/api/getColumnMapping.js

```diff
@@ -1,11 +0,0 @@
-'use strict';
-
-module.exports = (app) => {
```

Comment:
Strongly suggest keeping this. We'll need it for the next phase when we do syncing of multiple tables and or bases, When we handle multiple, mappings, system variables will no longer cut it and we will need a data entry method for a database of some sort